### PR TITLE
updated default configuration files GB-4618

### PR DESCRIPTION
**Pull request by fbjork**
# Description

* Updates the default SDL/TS config to use connectors instead of database models
* Updates the copy for newly initialized projects

# Type of change

- [ ] 💔 Breaking
- [ ] 🚀 Feature
- [ ] 🐛 Fix
- [ ] 🛠️ Tooling
- [ ] 🔨 Refactoring
- [ ] 🧪 Test
- [ ] 📦 Dependency
- [ ] 📖 Requires documentation update

### CODE REVIEW

File: cli/.prettierrc.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "semi": true,
+  "semi": false,
```
Comment (yoav-lavi) [inline cli/.prettierrc.json:2]: Why was this changed?
It'll likely case a lot of formatting changes if we have TS/JS files

Comment (fbjork) [reply cli/.prettierrc.json:2]: This should have been the default, this is what we have in the root config already and in other repos. I'll run formatting to ensure it's consistent.

Comment (yoav-lavi) [reply cli/.prettierrc.json:2]: There's other differences with this one (which is used for the CLI project), we could discuss using the same one but I'd do it separately personally

Comment (fbjork) [reply cli/.prettierrc.json:2]: Yes, will do it separately to ensure consistency across the mono repo.

File: cli/crates/backend/assets/grafbase.default.config.ts [hunk 2]
```diff
@@ -1,54 +1,40 @@
...
+// g.datasource(pg)
 
-  // Extend models with resolvers
-  // https://grafbase.com/docs/edge-gateway/resolvers
-  // gravatar: g.url().resolver('user/gravatar')
+// Resolvers - https://grafbase.com/docs/resolvers
+//
+g.query('helloWorld', {
+  returns: g.string(),
+  resolver: 'hello-world',
```
Comment (obmarg) [inline cli/crates/backend/assets/grafbase.default.config.ts:16]: Kind of wonder if we should have a `hello-world` resolver in the default output so that this works...

**Review by obmarg — APPROVED** [review]

File: cli/crates/backend/assets/grafbase.default.config.ts (hunk 2 above)
Comment (fbjork) [reply cli/crates/backend/assets/grafbase.default.config.ts:16]: Yeah I thought about this too, but didn't want to include any resolvers. by default. I could either comment this one out, but then the schema is empty. The error will only happen when you query the field.
